packages/cardano-services/src/Asset/TypeormAssetProvider/TypeormAssetProvider.ts

```diff
@@ -117,6 +117,7 @@ export class TypeormAssetProvider extends TypeormProvider implements AssetProvid
       if (!asset) throw new ProviderError(ProviderFailure.NotFound, undefined, `Asset not found '${assetId}'`);
       const supply = asset.supply!;
 
+      await queryRunner.release();
```

Comment:
Sorry for spotting this late, but it's a good practice to nest everything between `dataSource.createQueryRunner()` and `queryRunner.release()` in a `try` block and to call `queryRunner.release()` in the `finally` block.